tests/serverspec/default_spec.rb

```diff
@@ -9,31 +9,50 @@
 script_path = "/usr/sbin"
 scripts = %w(isakmpd_start)
 ssh_rc_command = "/etc/init.d/ssh"
+default_user = "root"
+default_group = "root"
+extra_user = ""
```

Comment:
`extra_user = "www"` and `extra_group = "www"` could remove the lines in "freebsd" and in "openbsd".

Reply:
is this a showstopper?

Reply:
the default value should be the largest common.

Reply:
although no answer, am assuming the answer is positive.

would you create a rule for it?